src/gen_fsm_compat.erl

```diff
@@ -638,55 +653,56 @@ get_msg(Msg) -> Msg.
 
 -if(?OTP_RELEASE >= 22).
 format_status(Opt, StatusData) ->
-	[PDict, SysState, Parent, Debug, [Name, StateName, StateData, Mod, _Time, _HibernateAfterTimeout]] =
-		StatusData,
-	Header = gen:format_status_header("Status for state machine",
-		Name),
+    [PDict, SysState, Parent, Debug, [Name, StateName, StateData, Mod,
+                                    _Time, _HibernateAfterTimeout]] =
+        StatusData,
+    Header = gen:format_status_header("Status for state machine", Name),
     Log = sys:get_log(Debug),
-	Specfic = format_status(Opt, Mod, PDict, StateData),
-	Specfic = case format_status(Opt, Mod, PDict, StateData) of
-				  S when is_list(S) -> S;
-				  S -> [S]
-			  end,
-	[{header, Header},
-		{data, [{"Status", SysState},
-			{"Parent", Parent},
-			{"Logged events", Log},
-			{"StateName", StateName}]} |
-		Specfic].
+    Specfic = format_status(Opt, Mod, PDict, StateData),
+    Specfic = case format_status(Opt, Mod, PDict, StateData) of
+                  S when is_list(S) -> S;
+                  S -> [S]
+              end,
+    [{header, Header},
+        {data, [{"Status", SysState},
+            {"Parent", Parent},
+            {"Logged events", Log},
+            {"StateName", StateName}]} |
+        Specfic].
 -elif(?OTP_RELEASE >= 21).
 format_status(Opt, StatusData) ->
-	[PDict, SysState, Parent, Debug, [Name, StateName, StateData, Mod, _Time, _HibernateAfterTimeout]] =
-		StatusData,
-	Header = gen:format_status_header("Status for state machine",
-		Name),
+    [PDict, SysState, Parent, Debug, [Name, StateName, StateData,
+                                      od, _Time, _HibernateAfterTimeout]] =
```

Comment:
The M is missing, of course `Mod` will be unbound. Please check every file again for this kind of error.

Reply:
I'm sorry. When I built it, everything worked.
I will check all files again, that such a mistake no longer occurs.